lmfit/lineshapes.py

```diff
@@ -302,6 +302,26 @@ def expsine(x, amplitude=1.0, frequency=1.0, shift=0.0, decay=0.0):
     return amplitude*sin(x*frequency + shift) * exp(-x*decay)
 
 
+def thermal_distribution(x, amplitude=1.0, center=0.0, kT=1.0, kind=-1):
+    """Return a thermal distribution function
+    kind = -1 (default) is the Bose-Einstein distribution
+    kind = 0 is the Maxwell-Boltzmann distribution
+    kind = 1 is the Fermi-Dirac distribution
+
+    thermal_distribution(x, amplitude=1.0, center=0.0, kBT=1.0, kind=-1):
```

Comment:
I suppose that ```kBT``` here should be ```kT``` instead?

Reply:
yes

Reply:
yes